pantalaimon/daemon.py

```diff
@@ -101,14 +101,15 @@ def __attrs_post_init__(self):
         accounts = self.store.load_users(self.name)
 
         for user_id, device_id in accounts:
+            token = False
             if self.conf.keyring:
                 try:
                     token = keyring.get_password(
                         "pantalaimon", f"{user_id}-{device_id}-token"
                     )
                 except RuntimeError as e:
                     logger.error(e)
-            else:
+            if not token:
```

Comment:
Is this needed? Isn't the `UseKeyring` option good enough? I'm not sure if I want to store access tokens unsafely if someone doesn't configure a keyring and has the `UseKeyring` option set to true. 

I don't feel very strongly about this so if you really think the config option isn't enough I'll merge this.

Reply:
The option is a better way - will use that.